### PR TITLE
Add physics info the upper-hybrid resonance docstring

### PR DESCRIPTION
**Pull request by alfkoehn**
Triggered by the hack-week, I just added a little bit of physics information about the upper-hybrid resonance (my research are is basically plasma-microwave interaction). Let me know all the details that I missed :)

Just realized that I need to

- add something to the changelog
- add my name to the author list

- [x] I have added a changelog entry for this pull request.

- [ ] If adding new functionality, I have added tests and
      docstrings.

- [ ] I have fixed any newly failing tests.

### CODE REVIEW

File: plasmapy/formulary/parameters.py [hunk 1]
```diff
@@ -1769,6 +1769,16 @@ def upper_hybrid_frequency(B: u.T, n_e: u.m ** -3) -> u.rad / u.s:
...
+    The upper hybrid frequency is a resonance for electromagnetic
+    waves in magnetized plasmas, namely for the X-mode. These are
+    waves with their wave electric field being perpendicular to 
+    the background magnetic field. In the cold plasma model, i.e.
+    without any finite temperature effects, the resonance acts
+    merely as a resonance such that power can be deposited there. 
+    If finite temperature effects are considered, mode conversion
+    can occur at the upper hybrid resonance, coupling to the 
+    electrostatic electron Bernstein wave. 
+
```
Comment (namurphy) [inline plasmapy/formulary/parameters.py:1781]:
```suggestion
    The upper hybrid frequency is a resonance for electromagnetic
    waves in magnetized plasmas, namely for the X-mode. These are
    waves with their wave electric field being perpendicular to
    the background magnetic field. In the cold plasma model, i.e.
    without any finite temperature effects, the resonance acts
    merely as a resonance such that power can be deposited there.
    If finite temperature effects are considered, mode conversion
    can occur at the upper hybrid resonance, coupling to the
    electrostatic electron Bernstein wave.

```
A minor suggestion to remove trailing white space.

**Review by namurphy — COMMENTED** [review]
So that's what the upper hybrid frequency is!  😹  The description is looking pretty close.  One point where I got a little confused was the beginning of the second sentence.  Does "these" correspond to X-mode waves?  And thanks!

**Comment by alfkoehn** [comment]
OK, I'm not exactly sure how to proceed: I've applied all the above listed changes (remove the trailing whitespaces, add a changelog, add my name to the contributors) and then pushed again. Do I just need to wait now for another review, since GitHub has recognized that I updated the branch (and thus also the pull request) ?

**Comment by StanczakDominik** [comment]
> OK, I'm not exactly sure how to proceed: I've applied all the above listed changes (remove the trailing whitespaces, add a changelog, add my name to the contributors) and then pushed again. Do I just need to wait now for another review, since GitHub has recognized that I updated the branch (and thus also the pull request) ?

Yes, unfortunately this is a recent change on the side on GitHub (actually a side effect of the crypto boom, of all things that we need to manually approve each test run for each new contributor. It's pretty terrible and I hope they adapt it to a toggle setting soon.

> Extra question: not sure how the list of contributors handles umlaute, I have one in my last name and used it here, is that a problem?

As a fellow unicode-named person, it's never been a problem with my ń, sphinx handles unicode without a hitch - and looking at https://plasmapy--1180.org.readthedocs.build/en/1180/about/credits.html it looks like it worked :)

File: changelog/1180.doc.rst [hunk 2]
```diff
@@ -0,0 +1,3 @@
+A brief description about the physics of the upper-hybrid resonance has been
+added to the docstring of the function upper_hybrid_frequency located in
+plasmapy/formulary/parameters.py
```
Comment (StanczakDominik) [inline changelog/1180.doc.rst:3]:
```suggestion
added to the docstring of the function `~plasmapy.formulary.parameters.upper_hybrid_frequency`.
```

This is a magic (as in sufficiently advanced technology) Sphinx trick that will link to the function's docstring from the changelog :)

**Review by StanczakDominik — APPROVED** [review]
The description looks great! I Sphinxified the changelog a little; I'll commit that to your branch and merge it :) Thanks, Alf!

**Comment by alfkoehn** [comment]
Looks like I have successfully managed to make my first contribution to PlasmaPy :)

**Pull request merged**